pkg/proxy/iptables/proxier.go

```diff
@@ -564,10 +564,17 @@ func (proxier *Proxier) syncProxyRules() error {
 				"-d", fmt.Sprintf("%s/32", externalIP),
 				"--dport", fmt.Sprintf("%d", info.port),
 			}
-			// We have to SNAT packets from external IPs.
+			// We have to SNAT packets to external IPs.
 			writeLine(rulesLines, append(args,
 				"-j", "MARK", "--set-xmark", fmt.Sprintf("%s/0xffffffff", iptablesMasqueradeMark))...)
-			writeLine(rulesLines, append(args,
+
+			externalTrafficOnlyArgs := append(args,
```

Comment:
Comment, something like

Allow traffic that is not from a bridge (i.e. not from a container) and not from a local process to be forwarded to the service.  This roughly translates to "all traffic from off-machine".  This is imperfect in the face of network plugins that might not use a bridge, but we can revisit that later.